wasmcloud-test-util/src/lib.rs

```diff
@@ -4,13 +4,65 @@ pub mod provider_test;
 #[cfg(not(target_arch = "wasm32"))]
 pub mod cli;
 
-// re-export testing interface
-pub use wasmcloud_interface_testing as testing;
-
 // re-export regex and nkeys
 pub use nkeys;
 pub use regex;
 
+#[allow(dead_code)]
+pub mod testing {
+
+    smithy_bindgen::smithy_bindgen!("testing/testing.smithy", "org.wasmcloud.interface.testing");
+
+    // after sdk is split we won't have to duplicate this code
+    impl Default for TestOptions {
+        fn default() -> TestOptions {
+            TestOptions {
+                patterns: vec![".*".to_string()],
+                options: std::collections::HashMap::default(),
+            }
+        }
+    }
+
+    pub type NamedResult<'name, T> = (&'name str, RpcResult<T>);
```

Comment:
Does this type alias really help making the code easier to understand and reason about? At the very least, it makes the tuple indexing below on L31 quite opaque